.env

```diff
@@ -21,6 +21,8 @@ REFRESH_ACCESS_TOKEN_ENDPOINT=null
 SEGMENT_KEY=null
 SITE_NAME=null
 USER_INFO_COOKIE_NAME=null
+USER_LOCATION_COOKIE_NAME=null
```

Comment:
This cookie is present in stage and production (checked the cookies in payment.edx.org) even though I did not see this added for payment MFE in `edx-internal` ([here](https://github.com/search?q=repo%3Aedx%2Fedx-internal%20USER_LOCATION_COOKIE_NAME&type=code))